Automattic/wp-calypso
# e2e: Switch to using twitter API for stability

#### Changes proposed in this Pull Request

* This PR changes how we check that the publicize functionality is working. Instead of loading the Twitter web client, we are now using the API. This will reduce the issues with rate limiting since we will only be making 1 call per check instead of 4+ per page load.

#### Testing instructions

* Ensure that tests pass in CI, especially interested in the test `Public Posts: Preview and Publish a Public Post`.
